examples/H3LAPD/2D-hw/square_quads.xml

```diff
@@ -0,0 +1,18 @@
nAr5tsPbsN8+8HtmJ8QvwPz7QfLMN9+sBzz7Qd3Yr794C7Mtx+swHz7wUr8ffvBKqy3H6zGevvBGqy3H6zFevvBOqy3H6xHbj+4G7n94B7k9oN7kdsP7kNuP9iA3J+NXK/5vonzxTfz+sXv5/1rHQ/w8xN/kJ+/+EP8/sQf5vcv/gifH/FH+fyJP8bnV3wLn3/xrXx/xLfx/RPfzvdX/HG+/+JPcP8Qf5L7j/hT3L/En+b+J/4M90/xZ7n/ij/H/Vv8ee7/4i/w/BDfwfNHfCfPL/EXef6Jv8TzU3wXz1/x3Ty/xffw/Bffex1PXrf9YB/x4Af7iQc/eJl48INXiAc/OEA8+MFB4sEPXkVuP3gN928/eB33Zz94A9dvP3gT8+0Hb2G+/eBtzLcfvIP5CfF3Md9+8B7m2w/ex3z7wQeYbz/4EPPtBx9hvv3gY/x9+8EhrLcfHMZ6+8EnWG8/+BTr7QefYb394HPk9oMjyO0HR5HbD44htx8cR24/+AK5P19yveb7Cc4XP8nrF/+K9691fM3PT/wUP3/xb/j9iX/L71/8Oz4/4qf5/Il/z+dX/Ac+/+I/8v0R/4nvn/jPfH/Ff+H7L/4r9w/xM9x/xM9y/xL/jfuf+O/cP8X/4P4r/if3b/Fz3P/Fz/P8EL/A80f8Is8v8Us8/8T/4vkp/jfPX/F/eH6L/8vzX/xy5Fq3/eAK8eAH/xEPfnCVePCDa8SDH6TIDjz4QUriwQ9SIbcfpCYe/CAN8eAHaYkHP0iH+faD9JhvP8iA+faDjJifEM+E+faDzJhvP8iC+faDrJhvP8iG+faD7JhvP8iBv28/yIn19oNcWG8/yI319oM8WG8/yIv19oN8yO0H+ZHbD25Abj8ogNx+UBC5/aAQcn8Kc73mexHOFy/K6xcvxvvXOorz8xMvwc9fvCS/P/FS/P7FS/P5ES/D5088wedXvCyff/FyfH/Ey/P9E6/A91e8It9/8UrcP8Qrc/8Rr8L9S7wq9z/xatw/xatz/xWvwf1bvCb3f/FaPD/Ea/P8Ea/D80u8Lq7fflCPePCD+rg/8wbEgx80xP3bDxpFrnXZDxoTD37QhHjwg6bEgx8kEQ9+0Ix48IPmxIMftEBuP2iJ+7cftML92Q9a4/rtB20w337QFvPtB+0w337QHvMT4h0w337QEfPtB50w337QGfPtB10w337QFfPtB93w9+0H3bHeftAD6+0HPbHeftAL6+0HvbHeftAHuf2gL3L7QT/k9oP+yO0HA5DbDwYi92cQ12u+D+Z88SG8fvGhvH+tYxg/P/Hh/PzFR/D7Ex/J7198FJ8f8dF8/sTH8PkVH8vnX3wc3x/x8Xz/xCfw/RWfyPdffBL3D/HJ3H/Ep3D/Ep/K/U98GvdP8encf8VncP8Wn8n9X3wWzw/x2Tx/xOfw/BKfy/NPfB7PT/H5PH/FF/D8Fl/I8198UeTKtR8sJh78YAnx4AdLiQc/uJF48IObiAc/uJl48INbkNsPbsX92w9uw/3ZD27H9dsP7sB8+8EyzLcfLMd8+8GdmJ8Qvwvz7QcrMN9+sBLz7QerMN9+sBrz7QdrMN9+sBZ/336wDuvtB+ux3n5wN9bbD+7BevvBvVhvP7gPuf1gA3L7wUbk9oNNyO0Hm5HbD+5H7s8DXK/5/iDniz/E6xd/mPevdTzCz0/8UX7+4o/x+xPfwu9ffCufH/FtfP7Et/P5FX+cz7/4E3x/xJ/k+yf+FN9f8af5/os/w/1D/FnuP+LPcf8Sf577n/gL3D/Fd3D/Fd/J/Vv8Re7/4i/x/BDfxfNHfDfPL/E9PP/E9/L8FN/H81d8P89v8Zd5/ou/Erl+135wgHjwg4PEgx+8Sjz4wWvEgx+8Tjz4wRvEgx+8idx+8Bbu337wNu7PfvAOrt9+8C7m2w/ew3z7wfuYbz/4APMT4h9ivv3gI8y3H3yM+faDQ5hvPziM+faDTzDffvAp/r794DOstx98jvX2gyNYbz84ivX2g2NYbz84jtx+8AVy+8GXyO0HJ5DbD04itx98hdyfr7le8/0U54t/w+sX/5b3r3V8x89P/DQ/f/Hv+f2J/8DvX/xHPj/iP/H5E/+Zz6/4L3z+xX/l+yN+hu+f+Fm+v+K/8f0X/537h/gf3H/E/+T+JX6O+5/4ee6f4he4/4pf5P4tfon7v/hfPD/E/+b5I/4Pzy/xf3n+iV/m+Sl+heev+H88v8Wv8vwXvxa56uwHKXIAD36Qknjwg1TEgx+kJh78IA3x4AdpiQc/SIfcfpCeePCDDLg/+0FGXL/9IBPm2w8yY779IAvm2w+yYn5CPBvm2w+yY779IAfm2w9yYr79IBfm2w9yY779IA/+vv0gL9bbD/Jhvf0gP9bbD27AevtBAay3HxREbj8ohNx+UBi5/aAIcvtBUeT2g2LI/SnO9ZrvJThfvCSvX7wU71/rKM3PT7wMP3/xBL8/8bL8/sXL8fkRL8/nT7wCn1/xinz+xSvx/RGvzPdPvArfX/GqfP/Fq3H/EK/O/Ue8Bvcv8Zrc/8Rrcf8Ur839V7wO92/xutz/xevx/BCvz/NHvAHPL/GGPP/EG/H8FG/M81e8Cc9v8aa4f/tBUuT63n7QjHjwg+bEgx+0IB78oCXx4AetiAc/aE08+EEb5PaDtrh/+0E73J/9oD2u337QAfPtBx0x337QCfPtB50xPyHeBfPtB10x337QDfPtB90x337QA/PtBz0x337QC3/fftAb6+0HfbDeftAX6+0H/bDeftAf6+0HA5DbDwYitx8MQm4/GIzcfjAEuf1gKHJ/hnG95vtwzhcfwesXH8n71zpG8fMTH83PX3wMvz/xsfz+xcfx+REfz+dPfAKfX/GJfP7FJ/H9EZ/M9098Ct9f8al8/8Wncf8Qn879R3wG9y/xmdz/xGdx/xSfzf1XfA73b/G53P/F5/H8EJ/P80d8Ac8v8YU8/8QX8fwUX8zzV3wJz2/xpTz/xf8Hn5YRlAAA</VERTEX>
```

Comment:
I think I still strongly prefer human readable data. These large binary blobs remind me of the xz backdoor. https://www.akamai.com/blog/security-research/critical-linux-backdoor-xz-utils-discovered-what-to-know